web/src/user_profile.js

```diff
@@ -324,6 +331,19 @@ export function get_custom_profile_field_data(user, field, field_types) {
     return profile_field;
 }
 
+function render_bot_profile_fields(bot) {
+    const user_type = {
+        name: bot.full_name,
+        should_add_guest_user_indicator: bot.is_guest,
+    };
+    $(".user_profile_name").html(render_user_full_name(user_type));
+
+    const $bot_owner_field = $(".bot_owner_user_field");
+    $bot_owner_field.attr("data-field-id", bot.bot_owner_id);
+
+    $("#avatar").css("background-image", `url(${people.medium_avatar_url_for_person(bot)})`);
```

Comment:
Do we need this separate code for bots? We anyways call this function on receiving `realm_user` event, so the code for humans can also handle bots.